src/index.html

```diff
@@ -7,7 +7,54 @@
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="stars">
+    <div class="container">
+      <ul class="stars stars--0">
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+      </ul>
+      <ul class="stars stars--1">
```

Comment:
Add empty lines between multiline sibling blocks of HTML. But don't add empty lines between parent and child elements. Fix it in all places, please.